lib/ukatomic/arch/arm/include/uk/asm/atomic.h

```diff
@@ -44,7 +44,7 @@
  * @return The index of the least significant 1-bit of x, or if x is zero,
  *   the result is undefined. ffs(1)=0, ffs(3)=0, ffs(0x80000000)=31
  */
-static inline unsigned int ukarch_ffs(unsigned int x)
+static inline unsigned int uk_ffs(unsigned int x)
```

Comment:
Actually these are bit ops, not atomic, so they should not had been part of `atomic.h` in the first place. Perhaps you should move these into their own `ukbitops` library and leave `ukatomic` with what is in `include/uk/atomic.h`.

Reply:
I think this should have it's own separate PR after this is merged